src/DIRAC/Resources/Computing/ARCComputingElement.py

```diff
@@ -79,9 +95,80 @@ def __init__(self, ceUniqueID):
 
     #############################################################################
 
+    def _isThereRestInterface(self):
+        # The name is the documentation. Importantly, if all is well the following variables are set
+        # - and a few others on which they (especially self.s) depend.
+        # self.rest    : Static variable to say if the REST interface is available and ready
+        # self.base_url: The basic URL for the REST interface
+        # self.s       : The python "requests" module session
+        # self.headers : The format of information we prefer : json of course
+        #
+        self.log.debug("Testing if the REST interface is available for %s" % self.ceHost)
+
+        # Is this needed? Test if the CE is supposed to be an ARC CE.
+        result = getCESiteMapping(self.ceHost)
+        if not result["OK"] or not result["Value"]:
+            self.log.error("Unknown CE ...")
+            return
```

Comment:
If we are here, `self.ceHost` should be valid, I think. So I would remove this check.